### PR TITLE
Use oc or kubectl to create rolebindings using new cli commands

### PR DESCRIPTION
**Pull request by lulf**

### CODE REVIEW

File: templates/install/deploy.sh [hunk 1]
```diff
@@ -188,13 +188,10 @@ fi
...
 runcmd "$CMD label -n ${NAMESPACE} sa enmasse-admin app=enmasse"
 
-if [ -n "$USE_OPENSHIFT" ]; then
-    runcmd "oc policy add-role-to-user view system:serviceaccount:${NAMESPACE}:default --rolebinding-name=enmasse-view" "Add permissions for viewing OpenShift resources to default user"
-    runcmd "oc policy add-role-to-user admin system:serviceaccount:${NAMESPACE}:enmasse-admin --rolebinding-name=enmasse-admin" "Add permissions for editing OpenShift resources to admin SA"
-
-    runcmd "oc label -n ${NAMESPACE} rolebinding enmasse-view app=enmasse"
-    runcmd "oc label -n ${NAMESPACE} rolebinding enmasse-admin app=enmasse"
-fi
+runcmd "$CMD create rolebinding -n ${NAMESPACE} enmasse-view --clusterrole=view --serviceaccount=${NAMESPACE}:enmasse-admin" "Create view role binding for enmasse-admin service account"
```
Comment (calohmn) [inline templates/install/deploy.sh]: I think the name `default-view` would fit better here.

File: templates/install/deploy.sh [hunk 2]
```diff
@@ -188,13 +188,10 @@ fi
...
+runcmd "$CMD create rolebinding -n ${NAMESPACE} enmasse-admin --clusterrole=admin --serviceaccount=${NAMESPACE}:enmasse-admin" "Create view role binding for enmasse-admin service account"
```
Comment (calohmn) [inline templates/install/deploy.sh]: Following the same convention, name would be `enmasse-admin-admin` here.

File: templates/install/deploy.sh [hunk 3]
```diff
@@ -234,11 +231,8 @@ if [ "$MODE" == "singletenant" ]; then
     runcmd "$CMD create -n ${NAMESPACE} -f $RESOURCE_DIR/plans/standard-plans.yaml" "Create standard address space plans"
     runcmd "$CMD create sa address-space-admin -n $NAMESPACE" "Create service account for default address space"
     runcmd "$CMD label -n ${NAMESPACE} sa address-space-admin app=enmasse"
-    if [ -n "$USE_OPENSHIFT" ]; then
-        runcmd "oc policy add-role-to-user admin system:serviceaccount:${NAMESPACE}:address-space-admin --rolebinding-name=enmasse-addressspace-admin" "Add permissions for editing OpenShift resources to address space admin SA"
-        runcmd "oc label -n ${NAMESPACE} rolebinding enmasse-addressspace-admin app=enmasse"
-    fi
-
+    runcmd "$CMD create rolebinding -n ${NAMESPACE} enmasse-addressspace-admin --clusterrole=admin --serviceaccount=enmasse:address-space-admin" "Create address space admin rolebinding"
```
Comment (calohmn) [inline templates/install/deploy.sh]: `--serviceaccount=${NAMESPACE}:address-space-admin"`

**Comment by lulf** [comment]
@calohmn Please have another look

**Pull request merged**